src/zenml/zen_server/deploy/helm/templates/server-db-job.yaml

```diff
@@ -101,6 +120,20 @@ spec:
               value: {{ $value | quote }}
             {{- end }}
             {{- end }}
+            {{- if .Values.zenml.proxy.enabled }}
+            - name: HTTP_PROXY
+              value: {{ .Values.zenml.proxy.httpProxy | quote }}
+            - name: HTTPS_PROXY
+              value: {{ .Values.zenml.proxy.httpsProxy | quote }}
+            - name: NO_PROXY
+              value: {{ include "zenml.noProxyList" . | quote }}
```

Comment:
Maybe I'm missing something, but I can't find where the `noProxyList` is defined. It should be a combination of `{{ .Values.zenml.proxy.noProxy }}` and `{{ .Values.zenml.proxy.additionalNoProxy }}`, correct?

Reply:
Yes, good point, I think I missed porting this over from the ZenML Pro chart.